client/src/interfaces.ts

```diff
@@ -105,3 +112,17 @@ export const emptyMediaResponse = (): MediaResponse => ({
   bytes: "",
   mimeType: "",
 });
+
+export interface GeoLocation {
```

Comment:
missing doc to explain the use of this interface

Reply:
👍 will update documentation by next PR